src/validator.js

```diff
@@ -93,29 +104,59 @@ exports.validate = function(xmlData, options) {
             tagFound = true;
             //continue; //text may presents after self closing tag
           } else {
-            return isValid;
+            //the result from the nested function returns the position of the error within the attribute
+            //in order to get the 'true' error line, we need to add the calculate the position where the attribute begins (i - attrStr.length) and then add the position within the attribute
```

Comment:
I believe you want to write "we need to calculate" instead of "we need to add the calculate"